README.md

```diff
@@ -96,13 +105,23 @@ A maintained and comprehensive list of awesome Fuel resources!
 - [Visualization of the Modular Ecosystem - @dabit3](https://twitter.com/dabit3/status/1551932518374539264)
 - [With rollups, we can build modular blockchain networks, removing the need to spin up new L1 smart contract platforms](https://twitter.com/musalbas/status/1582396496229507072?s=20&t=1KDBXVNLjF5NTk6C7gCj9Q)
 - [Writing dapps on Fuel is honestly so nice because you don't have to configure wagmi, hardhat/foundry, ethers.js... - @camiinthisthang](https://twitter.com/camiinthisthang/status/1577738478699171845)
+- [Introducing Sway Playground [Beta] by Nick Dodson](https://twitter.com/IAmNickDodson/status/1613573348172210177)
+- [Collection of projects within the Fuel ecosystem](https://twitter.com/fuelglobal_/status/1601218737981788160?s=46&t=kwX8f7P65LAXzWyN_FtKGA)
+- [Sway vs. Solidity examples](https://twitter.com/schwartzswartz/status/1617902570630156294)
+- [Sway Vectors in 6 tweets or less](https://twitter.com/jtriley_eth/status/1612539679085334528)
 
 ## Libraries
 - [Sway Libraries - Fuel Labs](https://github.com/sway-libs)
 - [Swaymigo - @sproule_/](https://github.com/BlockchainCap/swaymigo)
 
 ## Developer Resources 
 - [Sway frontend template implementation](https://github.com/brech1/sway-frontend)
+- [Web3RSVP](https://github.com/camiinthisthang/learnsway-web3rsvp)
```

Comment:
Let's not include anything until its 100% working and documented, there are still PRs open on this repo to get it working

Reply:
Removed this for now. Will move the repo to Fuel Labs org once it's ready and link to that fork in awesome-fuel